docs/pages/ds-audio_L3_Python_TestSpecification.md

```diff
@@ -0,0 +1,143 @@
+# Audio Settings L3 Python Test Specification
```

Comment:
merge these two documents, please, once document to describe the classes the .yaml files, then the same document to describe the tests and how it will work.

Reply:
done